ShareX.HelpersLib/Helpers/ImageHelpers.cs

```diff
@@ -223,6 +223,126 @@ public static Bitmap CropBitmap(Bitmap bmp, Rectangle rect)
             return null;
         }
 
+        public static Bitmap CutOutBitmapMiddleHorizontal(Bitmap bmp, int x, int width, CutOutEffectType effectType, int effectSize)
```

Comment:
You can add `Orientation orientation` parameter like in `CombineImages` function instead of using two different functions for horizontal and vertical. Because majority of codes are shared between these two functions.